spec/lib/bridgetown-inline-svg/render_optimized_svg_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "spec_helper"
+
+describe BridgetownInlineSvg::RenderOptimizedSvg do
+  let(:file_path) { source_dir("images", "square.svg") }
+  let(:attributes) { {} }
+  let(:instance_class) { described_class.new(file_path, attributes) }
+
+  it_behaves_like "RenderSvg"
```

Comment:
I'd really like to add some more tests to this one, what's the big advantage of the optomized svgs?

Reply:
View the code on these two svgs to see a good example:

https://gist.github.com/andrewmcodes/5beadd663eb09f5b773319dc9fca9283

Reply:
Awesome! I added a quick spec to check it removes the comment from the sample file, I think checking the library is definitely called & doing something should be about right.